### PR TITLE
[5.0] Remove the static execution of the module and add a text function

### PR DESCRIPTION
**Pull request by laoneo**
### Summary of Changes
Removes the static wrapper of module layout files. Like that they are executed in the context of the dispatcher. Additionally a text function is added so we are inline with [views](https://github.com/joomla/joomla-cms/pull/41079) and [plugins](https://github.com/joomla/joomla-cms/pull/41081).

### Testing Instructions
- Create any menu item
- Open the front end
- Click on the new menu item

### Actual result BEFORE applying this Pull Request
The breadcrumbs module shows the text "You are here".

### Expected result AFTER applying this Pull Request
The breadcrumbs module shows the text "You are here".

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: libraries/src/Dispatcher/AbstractModuleDispatcher.php [hunk 1]
```diff
@@ -134,4 +133,18 @@ protected function loadLanguage()
...
     }
+
+    /**
+     * Returns the string for the given key from the internal language object.
+     *
+     * @param   string  $key  The key
+     *
+     * @return  string
+     *
+     * @since   4.4.0
```
Comment (sandewt) [inline libraries/src/Dispatcher/AbstractModuleDispatcher.php:144]: `__DEPLOY_VERSION__` ?

Comment (laoneo) [reply libraries/src/Dispatcher/AbstractModuleDispatcher.php:144]: Yes

**Comment by sandewt** [comment]
@laoneo 
I see you've written quite a lot of code for J5. I'll do some PR tests after the holidays. If that is still necessary.

**Comment by laoneo** [comment]
Closing in favor of #41384.

**Pull request closed without merging**